bobanetwork/boba_legacy
# Feature (Gateway): Ve-Boba Lock for Vote page.

:clipboard: Closes, https://github.com/bobanetwork/ve-boba/issues/12

## Overview

Lock for vote page for ve boba

## Changes
- Added new page for lock
- Way to represent earlier, existing locks for the user
- Way to create new Lock
- Manage lock expired can be withdraw, other can have option to increase amount and extend time.
- Representation of voting power vs month in chart.

## Testing
- Navigate to lock page from Menu Items at top.
- Create new lock
- List of records or lock will be available as soon as lock has been created
- Pagination should work.
- Can have option to withdraw incase of expired lock.
- Can have option to increase amount or extend the expiry for existing lock which is yet to expired.
- Voting power and Voting ration will be calculated base on the selected expiry.
- Max expiry we have is 1yr.
